testutil/lowlevel.go

```diff
@@ -218,16 +228,35 @@ func (sys *SyscallRecorder) InsertFaultFunc(call string, fn func() error) {
 
 // Calls returns the sequence of mocked calls that have been made.
 func (sys *SyscallRecorder) Calls() []string {
-	return sys.calls
+	if len(sys.rcalls) == 0 {
+		return nil
+	}
+	calls := make([]string, 0, len(sys.rcalls))
+	for _, rc := range sys.rcalls {
+		calls = append(calls, rc.C)
+	}
+	return calls
 }
 
-// call remembers that a given call has occurred and returns a pre-arranged error, if any
-func (sys *SyscallRecorder) call(call string) error {
-	sys.calls = append(sys.calls, call)
-	if fn := sys.errors[call]; fn != nil {
-		return fn()
+// RCalls returns the sequence of mocked calls that have been made along with their results.
```

Comment:
What does the "R" stands for? Recorded?

Reply:
it used to be just CallResult so I made something short. RCall was it. It doesn't stand for anything I think.

Reply:
`stdlib` uses similar one-upcase-letter pattern, see [io.LimitedReader](https://godoc.org/io#LimitedReader)